unittest/libmariadb/bulk1.c

```diff
@@ -21,6 +21,13 @@
 
 static my_bool bulk_enabled= 0;
 
+#define SERVER_SUPPORT_BULK_UNIT_RESULTS(stmt)\
```

Comment:
better to pass mysql instead of stmt to the macro, so it could be used without creating a statement, e.g. to skip a test .